xhprof/callgrind_test.go

```diff
@@ -0,0 +1,53 @@
+package xhprof
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseCallgrind(t *testing.T) {
```

Comment:
You should add another test that is parsing the wordpress test data blob so we can verify that it works without errors. Write a few assertions for that.